tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+def validate_selenium(selenium_args):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    # Remove any browser repition.
+    browsers = list(set(selenium_args.split(',')))
+    for browser in browsers:
+        try:
+            # Create a class instance to see if everything is fine.
+            import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)().quit()
+        except:
```

Comment:
It's actually not always an `ImportError`. Sometimes this line would also give `Selenium`'s `WebDriverError` for example when you give a valid browser name (like chrome) but don't have the corresponding drivers installed.
The error message thus provided has two parts "Selenium specifications not valid"(for `ImportError`) and "corresponding WebDriver not installed" for `WebDriverError`.

I can break this into something like:
```python
except ImportError:
  msg = 'Selenium specifications not valid.'
except:
  msg = 'Corresponding WebDriver not installed.'
```
if that is more perferable